### ISSUE TITLE
Add fig.alt to vignette figures (a11y / pkgdown warnings)

### ISSUE THREAD

**Issue by adibender** [adibender/pammtools#276]
Building pkgdown surfaces several `Missing alt-text` accessibility warnings across the vignettes:

- `vignettes/baseline.Rmd`
- `vignettes/basics.Rmd`
- `vignettes/competing-risks.Rmd`
- `vignettes/convenience.Rmd`
- `vignettes/cumulative-effects.Rmd`
- `vignettes/data-transformation.Rmd`
- `vignettes/frailty.Rmd`
- `vignettes/left-truncation.Rmd`
- `vignettes/model-evaluation.Rmd`
- `vignettes/multi-state.Rmd`
- `vignettes/recurrent-events.Rmd`
- `vignettes/splines.Rmd`
- `vignettes/strata.Rmd`

Sweep PR: add `fig.alt = "..."` to plotting chunks. See `vignette(pkgdown::accessibility)`.
